### PR TITLE
Build action

### PR DESCRIPTION
**Pull request by GrantMoyer**
Adds build automation to ensure new PRs will not break the build. Also switches the unit testing framework to NUnit, since it's available for mono.

Eventually, the action could be extended to publish releases.

I created a new PR for this, since the existing one #15 wasn't triggering the GitHub workflow.

### CODE REVIEW

**Comment by GrantMoyer** [comment]
We may just have to merge this to get the actions to start running.

**Comment by patricknelson** [comment]
Assigning @dale-roberts since I'm only minimally capable of competently reviewing C# code (also time restricted). It's either that or tweak our branch settings. 😬

**Review by Robin-S — APPROVED** [review]

**Comment by GrantMoyer** [comment]
@patricknelson Since the stakes are pretty low here, I'm just going to disable requiring approvals for merges and trust us to do out due diligence before merging. If we make a mistake, that's what version control is for. (Plus hopefully we can have *some* automated quality control with GitHub actions)

Also, @Robin-S, since you've been active here lately, let me know if you'd like to be added to the MouseUnSnag org.

**Pull request merged**
